lib/ansible/modules/find.py

```diff
@@ -463,6 +463,12 @@ def statinfo(st):
         'isgid': bool(st.st_mode & stat.S_ISGID),
     }
 
+    if hasattr(st, 'st_blocks'):
```

Comment:
@Akasurde could you add test coverage for the `if not hasattr(st, 'st_blocks')`/`else` branch?